pkg/kubelet/dockershim/libdocker/kube_docker_client.go

```diff
@@ -152,18 +143,18 @@ func (d *kubeDockerClient) CreateContainer(opts dockertypes.ContainerCreateConfi
 func (d *kubeDockerClient) StartContainer(id string) error {
 	ctx, cancel := d.getTimeoutContext()
 	defer cancel()
-	err := d.client.ContainerStart(ctx, id)
+	err := d.client.ContainerStart(ctx, id, dockertypes.ContainerStartOptions{})
 	if ctxErr := contextError(ctx); ctxErr != nil {
 		return ctxErr
 	}
 	return err
 }
 
 // Stopping an already stopped container will not cause an error in engine-v1.
```

Comment:
The comment had `engine-api` but got replaced with `v1` in a bulk k8s api change. 
@yguo0905 could you confirm that the behavior is still the same with the new client and also fix the the comment?